bin/build

```diff
@@ -105,6 +105,10 @@ if [ -z "$NO_OPERATOR" ]; then
     cmd="operator-sdk build --image-build-args --pull $REPO/manageiq-operator:$TAG"
     echo "Building manageiq-operator: $cmd"
     $cmd
+
+    cmd="git rev-parse HEAD > ./build/_output/build_sha"
+    echo "Building manageiq-operator manifest: $cmd"
+    $cmd
```

Comment:
This has to come before the `operator-sdk build` otherwise it won't be in the container

Reply:
Oh I didn't realize the operator-sdk build also built the container...I thought it only built the binary

Reply:
Done...had to throw in a mkdir -p as well, as I was depending on the operator-sdk creating the dir.